benchmarks/src/main/scala/benchmarks/Benchmark.scala

```diff
@@ -23,7 +23,10 @@ case object ShortRunningTime    extends BenchmarkRunningTime(30000)
 case object UnknownRunningTime  extends BenchmarkRunningTime(1)
 
 abstract class Benchmark[T]() {
+  def preRun(): Unit = Unit
```

Comment:
We typically use `()` as unit value.

Reply:
Other frameworks call this kind of step `setup`. It seems to be a more natural name for this.